WebNetSample/Program.cs

```diff
@@ -2,14 +2,21 @@
 using Autofac.Extensions.DependencyInjection;
 using Business.Mappings;
 using WebNetSample.Business.DependencyResolvers.Autofac;
-using WebNetSample.Core.CrossCuttingConcerns.Caching.Microsoft;
+using WebNetSample.Core.DependencyResolvers;
+using WebNetSample.Core.Extensions;
+using WebNetSample.Core.Utilities.IoC;
 using WebNetSample.DataAccess;
 using WebNetSample.WebNetMVC.Middlewares;
 
 var builder = WebApplication.CreateBuilder(args);
 
 builder.Host.UseServiceProviderFactory(new AutofacServiceProviderFactory());
 
+builder.Services.AddDependencyResolvers(new ICoreModule[]
```

Comment:
Just want to clarify: why do we need this resolver?

Reply:
To add dependency resolvers for Core Layer actually. And to get services from ServiceTool.